.gitignore

```diff
@@ -35,3 +35,5 @@ examples/csharp/HelloPhi/models
 /src/java/.gradle
 /src/java/local.properties
 /src/java/build
+/src/java/CMakeFiles
+/src/java/CMakeCache.txt
```

Comment:
This is defined here:

https://github.com/microsoft/onnxruntime-genai/blob/c5745fd6d91a2ba2d94c6a52f6255652dde02508/src/java/CMakeLists.txt#L15

I *think* it's because `src/java/build` contains gradle build artifacts. This would help differentiate those two.